demosplan/DemosPlanCoreBundle/DoctrineMigrations/2023/10/Version20231006133151.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of the package demosplan.
+ *
+ * (c) 2010-present DEMOS plan GmbH, for more information see the license file.
+ *
+ * All rights reserved
+ */
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Exception;
+use Doctrine\DBAL\Platforms\MySQLPlatform;
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+class Version20231006133151 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'refs T34504: Create SupportContact Table and setup eMailAddress and customer relations';
+    }
+
+    /**
+     * @throws Exception
+     */
+    public function up(Schema $schema): void
+    {
+        $this->abortIfNotMysql();
+
+        $this->addSql('CREATE TABLE support_contact (id CHAR(36) NOT NULL, email_address CHAR(36) DEFAULT NULL, customer CHAR(36) DEFAULT NULL, title VARCHAR(255) DEFAULT NULL, phone_number VARCHAR(255) DEFAULT NULL, text LONGTEXT DEFAULT NULL, visible TINYINT(1) DEFAULT 0 NOT NULL, created_at DATETIME NOT NULL, updated_at DATETIME NOT NULL, UNIQUE INDEX UNIQ_F157EEC42B36786B (title), INDEX IDX_8C8C0928B08E074E (email_address), INDEX IDX_8C8C092881398E09 (customer), PRIMARY KEY(id)) DEFAULT CHARACTER SET UTF8 COLLATE `UTF8_unicode_ci` ENGINE = InnoDB');
```

Comment:
why should the title have a unique index?

Reply:
We changed this to a composite unique constraint over the fields customer and title in 431bc473b0162f224c720a38373462d2d605a37e